src/pages/projectSettings/tabs/GeneralTab/types.ts

```diff
@@ -2,7 +2,7 @@ import { ProjectType } from "../utils";
 
 export interface FormState {
   generalConfiguration: {
-    enabled: boolean | null;
+    enabled?: boolean | null;
```

Comment:
Since attached projects can no longer default to repo (the `null` value) this can just be
```suggestion
    enabled?: boolean;
```